src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
```

Comment:
:information_source: Missing detailed class-level documentation explaining the purpose and behavior of the ChatRequest class. (Severity: Minor)